Updated the get_extremes function

Emmanuel-Afrifa opened the pull request:
@lilyclements Please, I have updated the `get_extremes` function.

Please, in the issue, I think the operations for `between` and `outer` have been interchanged, so in the code, I did the reverse. Please, can you confirm if that's okay or if it was intentional, then I'll make the changes.

Also, I didn't utilize the `type` parameter as I didn't really understand the calculation between the percentile and threshold.

Lastly,  I didn't really know what `adjacent_column` meant, but I changed it to element as it was previously `"rainfall"`. Please, is that okay?

Thank you.

lilyclements commented:
@Emmanuel-Afrifa looks great! Good point on the "type" thing. We might add that in later, but for now I have removed it entirely. Can you pull the changes and run `devtools::document()` (otherwise I will later when I add in some examples and tests)

Emmanuel-Afrifa commented:
@lilyclements Well noted. I'll do that now.

Emmanuel-Afrifa commented:
@lilyclements Please, I have added the checks in the functions that didn't have them, and I may have changed a few things in the ones that had them. Please, can you verify that they are okay?

Also, I added them in the `summary_calculations` function. Is that okay?

Thank you.

lilyclements commented:
@Emmanuel-Afrifa can you commit (or revert) your changes made your side? Then try syncing?

lilyclements commented:
If that still doesn't work, then can you uncommit this most recent commit you made here? 

In RStudio, go to Terminal tab (bottom left pane) and run:

```
git log
```

Find the commit hash (looks like `a1b2c3d...`) of the bad commit. Then run:


```
git revert <commit_hash>
```

Git will open an editor for the commit message (or just auto-generate one). Save and close it. Finally run:

```
git push
```
